queue_processors/queue_processor/queueproc_utils/tests/test_messaging_utils.py

```diff
@@ -33,15 +124,17 @@ def test_send_pending_msg(self, mock_disconnect, mock_send, mock_connect, mock_c
         to the pending-queue, and finally disconnected from
         When: _send_pending_msg is called with a data argument
         """
-        MessagingUtils._send_pending_msg(self.test_data)
+        test_data = "data_dict"
+        pending_queue_name = '/queue/ReductionPending'
+        MessagingUtils()._send_pending_msg(test_data)
 
         mock_client.assert_called_once()
         mock_connect.assert_called_once()
         mock_send.assert_called_once()
         mock_disconnect.assert_called_once()
 
         (args, kwargs) = mock_send.call_args
```

Comment:
NIT, to align with the testing description docstring, should we assert the equality in a separate unit test?